services/apps/template_consumer/src/main.ts

```diff
@@ -0,0 +1,34 @@
+import { Config } from '@crowd/standard'
+import { ServiceConsumer, Options } from '@crowd/consumer'
+
+const config: Config = {
+  producer: {
+    enabled: false,
+    idempotent: true,
+  },
+  temporal: {
+    enabled: true,
+  },
+  redis: {
+    enabled: false,
+  },
+}
+
+const options: Options = {
+  maxWaitTimeInMs: 1000,
+  retryPolicy: {
+    initialRetryTime: 1000,
+    maxRetryTime: 1000,
+    retries: 1000,
```

Comment:
should this be configurable per service? maybe we need env variables for these. also retries being at 1000 feels like too much :D wouldn't this hinder processing power if activity/workflow has logical errors?